README.rst

```diff
@@ -82,6 +82,24 @@ On a Debian based system, the installation is as follows::
     make && make install
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+
+    git clone https://github.com/okbob/session_exec
+    cd session_exec
+    make USE_PGXS=1 && make USE_PGXS=1 install
```

Comment:
Ok, sudo is probably the way to go here, but it's not within the scope of this pull request. I was just fixing the missing documentation. If you guys want sudo, make another pull request for it. :pineapple: